### DISCUSSION TITLE
Printing to a ESCPOS shared printer in W10

### DISCUSSION THREAD

**Discussion by pruifernandes** [python-escpos/python-escpos#506] [Q&A] [answered]
Is it possible to print to a ESCPOS printer being shared by another pc (W10)?

PC1 has ESCPOS printer connected to it by USB. PC1 shares this printer.

Is it possible for a PC2 to print to this printer using this library?

**Comment by belono** [comment, marked as answer]
Hi @pruifernandes !

Yes, it is possible thanks to the `Win32Raw()` connector.
You can see [this](https://github.com/python-escpos/python-escpos/issues/443#issuecomment-767165052) reply.

Usage:
```
import escpos.printer as printer

p = printer.Win32Raw(printer_name='your_shared_printer_name')
p.open()
p.textln("Hello Win32Raw!")
p.close()
```
I think you need to install the [pywin32](https://github.com/mhammond/pywin32) package in order to use the `printer.Win32Raw() connector.`
Also, your shared printer must be able to print remotely from PC2, since the connector just forwards raw ESC/POS commands to the Windows print API.

**Reply by pruifernandes** [reply]
Thank you very much. Your solution is working well for me!
